File: Fase 2/revistas-app/src/app/objetos/editor/PrecioSuscripcion.ts
export class PrecioSuscripcion {
  private _idRevista: number;
  private _fechaInicio: Date;
  private _precio: number;

  constructor(idRevista: number, fechaInicio: Date, precio: number) {
    this._idRevista = idRevista;
    this._fechaInicio = fechaInicio;
    this._precio = precio;
  }

  public get idRevista() {
    return this._idRevista;
  }

  public get fechaInicio() {
    return this._fechaInicio;
  }

  public get precio() {
    return this._precio;
  }
}

File: Fase 2/ServidorRevistas/src/main/java/objetos/PrecioSuscripcion.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package objetos;

import java.sql.Date;

/**
 *
 * @author fernanrod
 */
public class PrecioSuscripcion {
    private Date fechaInicio;
    private Float precio;
    private Integer idRevista;
    
    public PrecioSuscripcion(Date fechaInicio, Float precio, Integer idRevista) {
        this.fechaInicio = fechaInicio;
        this.precio = precio;
        this.idRevista = idRevista;
    }
    
    public Date getFechaInicio() {
        return fechaInicio;
    }
    
    public Float getPrecio() {
        return precio;
    }
    
    public Integer getIdRevista() {
        return idRevista;
    }
}
